### PR TITLE
[DO-2077] Check permission for export

### PR DESCRIPTION
**Pull request by Iva-khar**

### CODE REVIEW

File: src/components/table/ExportXlsx.jsx [hunk 1]
```diff
@@ -13,9 +13,21 @@ const ExportXlsx = (props) => {
...
   const exportXlsxModalRef = useRef();
+  const [hasPermission, setHasPermission] = useState(false);
 
   const openExportXlsxModal = () => exportXlsxModalRef.current.show();
 
+  const checkPermission = () => {
+    Api.get('payment/project/')
+      .then((resp) => {
+        resp.data.forEach((project) => {
+          if (project.status === 'active' && project.is_active && project.active_subscription.price > 0) {
```
Comment (roman-tiukh) [inline src/components/table/ExportXlsx.jsx:24]: I suggest calculate this one time on the backend and save in Redux in user data.
`DataOceanUserSerializer`

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: src/components/table/ExportXlsx.jsx (hunk 1 above)
Comment (Iva-khar) [reply src/components/table/ExportXlsx.jsx]: done

File: src/components/table/ExportXlsx.jsx [hunk 2]
```diff
@@ -24,6 +25,20 @@ const ExportXlsx = (props) => {
     exportXlsxModalRef.current.hide();
   };
 
+  if (!useSelector((store) => store.user.paid_subscription_permission)) {
```
Comment (roman-tiukh) [inline src/components/table/ExportXlsx.jsx:28]: hooks should be declared at top of component

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: src/components/table/ExportXlsx.jsx (hunk 2 above)
Comment (Iva-khar) [reply src/components/table/ExportXlsx.jsx]: done

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**